rusoto/services/cur/src/generated.rs

```diff
@@ -82,22 +81,14 @@ pub struct ReportDefinition {
     #[serde(rename = "AdditionalArtifacts")]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub additional_artifacts: Option<Vec<String>>,
-    #[serde(rename = "AdditionalSchemaElements")]
-    pub additional_schema_elements: Vec<String>,
-    #[serde(rename = "Compression")]
-    pub compression: String,
-    #[serde(rename = "Format")]
-    pub format: String,
-    #[serde(rename = "ReportName")]
-    pub report_name: String,
-    #[serde(rename = "S3Bucket")]
-    pub s3_bucket: String,
-    #[serde(rename = "S3Prefix")]
-    pub s3_prefix: String,
-    #[serde(rename = "S3Region")]
-    pub s3_region: String,
-    #[serde(rename = "TimeUnit")]
-    pub time_unit: String,
+    #[serde(rename = "AdditionalSchemaElements")] pub additional_schema_elements: Vec<String>,
```

Comment:
Nit: this seems like a `rustfmt` regression. What does `$ rustup run nightly rustc --version` show for you? 

I get `rustc 1.25.0-nightly (97520ccb1 2018-01-21)` and it generates the two line version.